ddl/backfilling.go

```diff
@@ -596,7 +597,11 @@ func handleOneResult(result *backfillResult, scheduler backfillScheduler, consum
 		scheduler.drainTasks() // Make it quit early.
 		return result.err
 	}
-	*totalAddedCount += int64(result.addedCount)
+	if result.totalCount > 0 {
```

Comment:
If this means totalCount is zero or negtive number, we treat it as init value and it should be zero?